documentation/src/docs/beginner_dev_guide.md

```diff
@@ -33,14 +37,17 @@ To complete this guide, you need:
   
 * An IDE
   
-* JDK 11+ installed with JAVA_HOME configured appropriately
-  
-* <details>
-  <summary>Gradle 6.5.1+</summary>
-  Installation : https://gradle.org/install/
+* JDK 11+ installed with `JAVA_HOME` configured appropriately
```

Comment:
For Java code snippets, can they be formatted with colors? Maybe it is clearer for most people, similar to how the editor renders the code.

Reply:
Java and Groovy syntax highlighting was restored here: https://github.com/pravega/docs/pull/1/commits/8d04d611d32f77607f8fb42f671bce22b49d9a37#diff-037321d390c744ebdb9fd2e384a47923718d2228532e7127bdbc1baacda55597